index.html

```diff
@@ -2810,6 +2873,31 @@ <h2 id="View">Backbone.View</h2>
       you want to wait to define them until runtime.
     </p>
 
+    <p id="View-preinitialize">
+      <b class="header">preinitialize</b><code>new View([options])</code>
+      <br />
+      For ES2015 classes. If you define a <b>preinitialize</b> method, it will be invoked when
+      the View is first created and before any instantiation logic is run for the View.
+    </p>
+
+<pre>
+class Document extends Backbone.View {
+    preinitialize() {
+      this.el: "body",
```

Comment:
These should be `=` because they're variable assignments not properties.

Though I'm not sure why this can't be put on the class directly. Maybe use a different example for this?